### PR TITLE
Refactor inventory mgmt again

### PR DESCRIPTION
**Pull request by planteater**
Take another crack at this.

**This PR Addresses:**  
[Another horrible bug!](https://trello.com/c/icPu2wJW/492-another-horrible-bug)

### CODE REVIEW

File: inventory.js [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+import {
+	commands,
+	DecorationRangeBehavior,
+	Position,
+	Range,
+	Selection,
+	TextDocumentShowOptions,
+	TextEditor,
+	TextEditorRevealType,
+	Uri,
```
Comment (planteater) [inline inventory.js:12]: asdfdsaf dsa asdf sadf sdf sdf dsf

File: tracking.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+export class ProtocolHandler implements UriHandler {
+	private disposable: Disposable;
+
+	constructor() {
+		this.disposable = window.registerUriHandler(this);
+	}
+
+	async beerHandleUri(uri: Uri) {
+		await Container.webview.handleProtocol(uri);
+	}
```
Comment (planteater) [inline tracking.js:14]: 3 fr 3

File: foo.js [hunk 3]
```diff
@@ -10,9 +10,25 @@ import { Container } from "./container";
...
 
+export interface ShowReviewDiffCommandArgs {
+	reviewId: string;
+	checkpoint: CSReviewCheckpoint;
+	repoId: string;
+	path: string;
+}
+
+export interface ShowReviewLocalDiffCommandArgs {
+	repoId: string;
```
Comment (planteater) [inline foo.js:24]: path foo

**Comment by planteater** [comment]
out of change set

```
export interface ShowReviewDiffCommandArgs {
	reviewId: string;
	checkpoint: CSReviewCheckpoint;
	repoId: string;
	path: string;
}
```
								
[foo.js](https://github.com/codestreamtest/beerfridge/blob/426c916107e3556e85ba3b5137d224aed52c40e8/foo.js#L38-L43) (Lines 38-43)

**Review by planteater — COMMENTED** [review]
finish it

File: foo.js [hunk 4]
```diff
@@ -10,9 +10,25 @@ import { Container } from "./container";
...
 const command = createCommandDecorator(commandRegistry);
 
+export interface ShowReviewDiffCommandArgs {
+	reviewId: string;
+	checkpoint: CSReviewCheckpoint;
+	repoId: string;
+	path: string;
+}
+
+export interface ShowReviewLocalDiffCommandArgs {
```
Comment (planteater) [inline foo.js:23]: Not so good

**Comment by planteater** [comment]
you forgot this...

```
export interface ShowMarkerDiffCommandArgs {
	marker: CSMarkerIdentifier;
}
```
								
[foo.js](https://github.com/codestreamtest/beerfridge/blob/426c916107e3556e85ba3b5137d224aed52c40e8/foo.js#L49-L51) (Lines 49-51)

**Review by planteater — COMMENTED** [review]
good job

**Review by codestreamtest — APPROVED** [review]
All good in the hood.

**Pull request merged**